brabbuss/rancid-tomatillos
# Navbar

#### What does this PR do?
- `Navbar` provides basic navigation and a searchbar for later implementation
- `Banner` is a carousel of hero images that from three random movies with accompanying text and link to that movie
- `Banner` utilizes object destructuring to store three random choices every time the movie page is loaded
- Add a proper title to the web page
- Bring `Movies` component back into `App`, migrate `MovieCard` back into `Movies`, add `Banner` to `Movies` so that it will only render on the home page - change from class component to stateless functional component

#### How should this be manually tested?
- Opening the site, you should be presented with a hero image carousel, each image should click through to the details of that movie
- should always see the navbar at the top of the site. clicking 'Rancid Tomatillos' will take you back home
- Nav should be responsive
- other links and search are not currently functional

#### Any background context you want to provide?
- Needed to add `Movies` component back in to function as a container for `MovieCard` and `Banner`

#### What are the relevant tickets?
closes #21
closes #31
